php/ext/google/protobuf/protobuf.c

```diff
@@ -33,6 +33,8 @@
 #include <php.h>
 #include <Zend/zend_interfaces.h>
 
+#include "php_protobuf.h"
```

Comment:
Why does this need to be included here? Will one of the other macros reference `phpext_protobuf_ptr`?

Reply:
Oops, Deleted. Thanks